kythe/extractors/openjdk11/extract/extract.go

```diff
@@ -296,5 +296,7 @@ func logCollectedErrors(stderr io.Reader) {
 	if err != nil {
 		log.Println(err)
 	}
-	log.Println("Error extracting modules:\n\t", strings.Join(errors, "\n\t"))
+	if len(errors) > 0 {
+		log.Println("Error extracting modules:\n\t", strings.Join(errors, "\n\t"))
```

Comment:
Are format strings supported for Println (not printf)?

Reply:
never mind, you didn't use a format string

Reply:
No, but they also aren't in use here.